apps/accounts/views/mfa.py

```diff
@@ -13,6 +13,7 @@
 from ...utils import get_client_ip
 import sys
 from django.views.decorators.cache import never_cache
+from axes.decorators import axes_dispatch
```

Comment:
Is there a reason you decided to go with Axes instead of DRF? Not a concern, just a question

Reply:
Yup, good question. Axes lets you control the number of *failed* login attempts for a username before locking the account. DRF and other throttling or ratelimiting tools let you control, more generally, how quickly or at what volume an IP, or token, or user, can make requests.